### PR TITLE
chore: resolvers settings file and update core lib

### PR DESCRIPTION
**Pull request by martinsaporiti**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -24,8 +24,8 @@ require (
 	github.com/iden3/contracts-abi/state/go/abi v1.0.2-0.20231006071010-f511d08ca36f
 	github.com/iden3/go-circuits/v2 v2.3.0
 	github.com/iden3/go-iden3-auth/v2 v2.4.1
-	github.com/iden3/go-iden3-core/v2 v2.2.0
-	github.com/iden3/go-iden3-crypto v0.0.16
+	github.com/iden3/go-iden3-core/v2 v2.3.1
```
Comment (OBrezhniev) [inline go.mod:27]: please update other iden3 deps too

File: resolvers_settings_sample.yaml [hunk 2]
```diff
@@ -31,14 +71,119 @@ polygon:
     rpcResponseTimeout: 5s
     waitReceiptCycleTime: 30s
     waitBlockCycleTime: 30s
-    gasLess: false
+    gasLess: true
```
Comment (OBrezhniev) [inline resolvers_settings_sample.yaml:74]: it shouldn't be gasLess

File: resolvers_settings_sample.yaml [hunk 3]
```diff
@@ -14,11 +14,51 @@ polygon:
     gasLess: true
     rhsSettings:
       mode: None
-      contractAddress: 0x16A1ae4c460C0a42f0a87e69c526c61599B28BC9
+      contractAddress: 0x3d3763eC0a50CE1AdF83d0b5D99FBE0e3fEB43fb
```
Comment (OBrezhniev) [inline resolvers_settings_sample.yaml:17]: On all networks rhs should be 0x7dF7...

**Review by OBrezhniev — CHANGES_REQUESTED** [review]

File: resolvers_settings_sample.yaml (hunk 3 above)
Comment (OBrezhniev) [reply resolvers_settings_sample.yaml:17]: Also I don't remember if we have new rhs url for offchain.

Comment (martinsaporiti) [reply resolvers_settings_sample.yaml]: I think we dont have a new domain for rhs.

File: resolvers_settings_sample.yaml [hunk 4]
```diff
@@ -34,11 +74,116 @@ polygon:
...
+    defaultGasLimit: 600000
+    confirmationTimeout: 10s
+    confirmationBlockCount: 5
+    receiptTimeout: 600s
+    minGasPrice: 0
+    maxGasPrice: 1000000
+    rpcResponseTimeout: 5s
+    waitReceiptCycleTime: 30s
+    waitBlockCycleTime: 30s
+    gasLess: true
```
Comment (OBrezhniev) [inline resolvers_settings_sample.yaml:136]: false

File: resolvers_settings_sample.yaml [hunk 5]
```diff
@@ -34,11 +74,116 @@ polygon:
...
```
Comment (OBrezhniev) [inline resolvers_settings_sample.yaml:95]: gasLess: false

File: resolvers_settings_sample.yaml [hunk 6]
```diff
@@ -34,11 +74,116 @@ polygon:
...
```
Comment (OBrezhniev) [inline resolvers_settings_sample.yaml:177]: false

**Review by OBrezhniev — CHANGES_REQUESTED** [review]

**Review by OBrezhniev — APPROVED** [review]

**Pull request merged**